src/crawler/crawlers/arso_daily_weather/crawler.js

```diff
@@ -0,0 +1,247 @@
+// imports
+const CrawlerUtils = require('../../utils/crawlerutils');
+const SQLUtils = require('../../utils/sqlutils');
+
+// external imports
+const cheerio = require('cheerio'); // handling DOM
+const moment = require('moment'); // handling time
+
+
+/**
+ * Class for template crawler.
+ */
+class TemplateCrawler {
+    /**
+     * Responsible for loading state
+     */
+    constructor() {
+        // loading config
+        this.config = CrawlerUtils.loadConfig(__dirname);
+        // loading state
+        this.state = CrawlerUtils.loadState(__dirname);
+
+        this.SQLUtils = new SQLUtils();
+    }
+
+    /**
+     * Responsible for crawling at one step.
+     *
+     * NOTE: Take care that the crawler is resistant to delays (it crawls
+     * also historic data if needed). Crawler should provide all the steps
+     * denoted in the comments of this method.
+     */
+    async crawl() {
+        console.log('Starting crawl: ' + this.config.id);
+        // do the crawling here
+        const weatherData = await this.getData();
+
+
+        // update datalake repository with the crawled data
+        // update the state with the last crawled timestamp
+        await this.dataLake(weatherData);
+
+
+        // pushing to db
+
+        // existing metadata
+        const nodes = await this.SQLUtils.getNodes();
+        let sensors = await this.SQLUtils.getSensors();
+        // console.log(sensors)
+
+        // inserting nodes and sensors
+        for (const station of weatherData) {
+            const node = nodes.filter((x) => x.uuid === station.Postaja);
+            let el = 0;
+            if (node.length === 0) {
+                await this.SQLUtils.insertNode(station.Postaja, station.Postaja, `${station.Postaja} - samodejna postaja`);
+            };
+
+            const sensor = sensors.filter((x) => x.node_uuid === station.Postaja);
+
+            if (sensor.length === 0) {
+                let i = 0;
+                for (const stationSensors of Object.entries(station)) {
```

Comment:
We prefer usage of Object.keys. So you later do not have to operate with stationSensor[0] and stationSensor[1], but you can name this as key and value.